lib/Qpsmtpd/Plugin.pm

```diff
@@ -347,10 +347,16 @@ sub _register_standard_hooks {
     }
 }
 
+sub db_args {
+    my ( $self, %arg ) = @_;
```

Comment:
* We know that that errors in the logs can go unnoticed for a long time
* it's a programming error we want raised early, preferably early enough that it's caught in testing.
* we don't want a plugin failure to take down the mail server, but IIRC, plugin exceptions are handled.